plugin/CurtinIncSw.vim

```diff
@@ -1,17 +1,27 @@
 function! FindInc()
   let dirname=fnamemodify(expand("%:p"), ":h")
-  let cmd="find . " . dirname . " -type f -iname " . t:IncSw . " | head -n1 | tr -d '\n'"
-
-  let findRes=system(cmd)
+  let findRes=""
+  let targetFile=b:IncSw
+  for location in [dirname, '.' ]
```

Comment:
If recursively searching the current directory is a concern, perhaps we could set the max-depth?